docker-compose.yml

```diff
@@ -1,12 +1,14 @@
 version: '2'
 services:
   autopush:
-    image: bbangert/autopush
+    # image: bbangert/autopush
+    build: .
     hostname: autopush
     environment:
       - LOCAL_HOSTNAME=localhost
       - ROUTER_HOSTNAME=autopush
       - AWS_LOCAL_DYNAMODB=http://dynamodb:8000
+      - AWS_LOCAL_CREDS=Bogus:Bogus
```

Comment:
one more thing, forgot to kill it here and below (possibly in the commit message too?)